tests/test_go.py

```diff
@@ -67,19 +67,22 @@ def test_build_kured(auto_container_per_test, container_git_clone):
     [
         GitRepositoryBuild(
             repository_url="https://github.com/helm/helm.git",
-            repository_tag="v3.14.2",
-            build_command="make build test-unit",
+            repository_tag="v3.14.3",
+            build_command="make build && make test-unit",
```

Comment:
where's the difference between the two? as we don't run with "-j" it should be the same?

Reply:
actually none, I thought it would fix the test issues before looking closer